site/docs/hosting/comparison.md

```diff
@@ -0,0 +1,62 @@
+# Comparison of Hosting Providers
+
+There are many different hosting providers that allow you to run your bot.
+It can sometimes be hard to keep track of how much they cost and how good their performance is.
+This is why the grammY community is collecting their experiences on this page.
+
+## What is a Hosting Provider?
+
+In order to keep a bot online 24 hours a day, you need to run a computer 24 hours a day.
+As [mentioned in the introduction](/guide/introduction.html#how-to-keep-a-bot-running), you most likely don't want to do that with your laptop or home computer.
+Instead, you can ask a company to run the bot in the cloud.
+
+In other words, you just run it on someone else's computer.
+
+## Comparison Tables
+
+> Please click the edit button at the bottom of the page to add more providers or to edit existing ones!
+
+We have two comparison tables, one for [serverless](#what-does-serverless-mean) hosting and one for [VPS](#vps).
+
+### Serverless
+
+Serverless means that you do not control a single machine on which your bot is run.
+Instead, these hosting providers will rather allow you to upload your code, and then start and stop different machines as necessary to make sure that your bot always works.
+
+This has the downside that your bot does not have access to a persistent storage by default, such as a local file system.
+Instead, you will often have to have a database separately and connect to it if you need to store data permanently.
+We therefore recommend you to use a different kind of hosting for more complex bots, e.g. a [VPS](./vps.md).
+
+The main thing to know about them is that you must run your bot on webhooks on serverless infrastructure.
+
+| Name                  | Min. price | Limits                                                                                             | Ping and Location                     | Node.js            | Deno               | Web                | Notes                                |
+| --------------------- | ---------- | -------------------------------------------------------------------------------------------------- | ------------------------------------- | ------------------ | ------------------ | ------------------ | ------------------------------------ |
+| Vercel                | Free       | [Unlimited invocations, 100 GB-hours, 10s time limit](https://vercel.com/pricing)                  | $20/mo subscription                   | :question:         | :question:         | :question:         | Not intended for non-websites?       |
```

Comment:
What does 100 "GB-hours" mean? 100 GB per hour?

Reply:
No. GB-hours are a unit that describe space over a period of time. For example, one GB-hour would be storing 1 GB of data for one hour, or 1 MB of data for 1024 hours, or 1 TB of data for $1/1024$ hours.